### PR TITLE
Module 4 solution to javascript problem

### PR DESCRIPTION
**Pull request by mareliefi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Module 4 Solution Starter</title>
+  <script>
```
Comment (HAASLEWER) [inline index.html:9]: Please move script tags to right before the body closing tag.

File: SpeakGoodBye.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+(function (window) {
+  var byeSpeaker = {};
+  var speakWord = "Good Bye";
```
Comment (HAASLEWER) [inline SpeakGoodBye.js]: Use `const` for constants: https://www.w3schools.com/js/js_const.asp

File: SpeakHello.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+(function (window) {
+  var helloSpeaker = {};
+  var speakWord = "Hello";
```
Comment (HAASLEWER) [inline SpeakHello.js]: Use `const`

**Review by HAASLEWER — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (mareliefi) [reply index.html:9]: Done thank you

File: SpeakGoodBye.js (hunk 2 above)
Comment (mareliefi) [reply SpeakGoodBye.js]: Noted, thank you

File: SpeakHello.js (hunk 3 above)
Comment (mareliefi) [reply SpeakHello.js]: Noted, thank you

**Review by HAASLEWER — APPROVED** [review]

**Pull request merged**